packages/utils/node-resolver-core/src/NodeResolver.js

```diff
@@ -275,8 +275,13 @@ export default class NodeResolver {
       };
     }
 
-    if (!this.shouldIncludeNodeModule(env, filename)) {
+    let shouldInclude = this.shouldIncludeNodeModule(env, filename);
+    if (!shouldInclude) {
       return null;
+    } else if (typeof shouldInclude === 'string') {
```

Comment:
Is this all that's needed to allow support for globals?

Reply:
Yes (so theoretically, you could create a resolver plugin that does this - without any changes to Parcel itself).

Reply:
Alright. I thought adding globals support as a core feature would be useful, which is why I made this PR, but if it's out of scope for the project I could remove it and make a plugin. Which is preferable?

Reply:
We are not against more features, but renaming `includeNodeModules` is a rather breaking change.

Reply:
I was hoping that breaking changes wouldn't be a major annoyance since Parcel v2 is still in beta and not that many people are using it in comparison to v1. If necessary I'll create a plugin, but I don't like the idea of fragmenting similar functionality into separate places, especially when globals should be configurable on a target-by-target basis.